SrfGen/createSRF.py

```diff
@@ -1175,7 +1175,7 @@ def CreateSRF_multi(
                 "bot_taper=0.02",
                 "top_taper=0.0",
                 "{}={}".format(rup_name, rup_delay[c]),
-                "srf_version=1.0",
+                "srf_version=2.0",
```

Comment:
I'm pretty sure that this doesn't work. Because our SRF reader can't read version 2 SRFs. 3.3 only returns version 1 SRFs

Reply:
We can read some v2 srfs, they just have to have a single point block like v1